.github/workflows/test-minimal-deps.yml

```diff
@@ -39,14 +39,5 @@ jobs:
       - run: pip install .
       - run: pip install pytest pytest-cov
       - run: pytest --cov=fairlearn --cov-report=xml test/install
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
-        if: ${{ (inputs.codeCovPython == inputs.python) && (steps.get_date.outputs.AGE < 172800) }} # 172800 = 48 * 3600
```

Comment:
I think this conditional is needed, or our test matrix ends up sending too many results to CodeCov. At least for any build which runs regularly.

Reply:
Got it! I don't have experience with the service prior to this, so I wasn't sure how it is exactly used in this context. I will bring it back.